editor.planx.uk/src/pages/Team/components/AddFlow/types.ts

```diff
@@ -1,3 +1,4 @@
+import { boolean } from "mathjs";
```

Comment:
Is the right import? I'd expect `import { number, object, string, boolean } from "yup";` ?

Reply:
Good spot! I've auto imported the wrong thing here